src/locutus/api/sessions.py

```diff
@@ -2,6 +2,7 @@
 from flask import request
 from locutus.model.exceptions import *
 from locutus.api import get_editor, default_headers
+import pdb
```

Comment:
You can remove this once you have identified the problem. If a set_trace statement makes its way into google, it can cause some really confusing behavior (while it waits forever for you to interact with it)